packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -509,7 +585,7 @@ class SelectableRegionState extends State<SelectableRegion> with TextSelectionDe
   /// If the selectable subtree returns a [SelectionResult.pending], this method
   /// continues to send [SelectionEdgeUpdateEvent]s every frame until the result
   /// is not pending or users end their gestures.
-  void _triggerSelectionEndEdgeUpdate() {
+  void _triggerSelectionEndEdgeUpdate({TextGranularity? textGranularity}) {
```

Comment:
Should there be spaces between the brackets and the text?  I forget.  Here and below, if I'm right.

Reply:
Looks like the other methods in this file with named parameters do not, so I just followed that. But I am also unsure.

Reply:
Sounds like it's not a big deal then, keeping it like this is fine :+1: